app/org/sagebionetworks/bridge/models/accounts/UserConsentHistory.java

```diff
@@ -0,0 +1,185 @@
+            return this;
+        }
+        public Builder withBirthdate(String birthdate) {
+            this.birthdate = birthdate;
+            return this;
+        }
+        public Builder withImageData(String imageData) {
+            this.imageData = imageData;
+            return this;
+        }
+        public Builder withImageMimeType(String imageMimeType) {
+            this.imageMimeType = imageMimeType;
+            return this;
+        }
+        @JsonDeserialize(using = DateTimeToPrimitiveLongDeserializer.class)
+        public Builder withSignedOn(long signedOn) {
+            this.signedOn = signedOn;
+            return this;
+        }
+        @JsonDeserialize(using = DateTimeToLongDeserializer.class)
+        public Builder withWithdrewOn(Long withdrewOn) {
+            this.withdrewOn = withdrewOn;
+            return this;
+        }
+        public Builder withHasSignedActiveConsent(boolean hasSignedActiveConsent) {
+            this.hasSignedActiveConsent = hasSignedActiveConsent;
+            return this;
+        }
+        public UserConsentHistory build() {
+            // validation not necessary, this is only created internally.
+            return new UserConsentHistory(healthCode, studyIdentifier, consentCreatedOn, name,
+                    birthdate, imageData, imageMimeType, signedOn, withdrewOn, hasSignedActiveConsent);
+        }
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(healthCode, studyIdentifier, consentCreatedOn, name, birthdate, imageData, imageMimeType,
+                signedOn, withdrewOn, hasSignedActiveConsent);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null || getClass() != obj.getClass())
+            return false;
+        UserConsentHistory other = (UserConsentHistory) obj;
+        return (Objects.equals(healthCode, other.healthCode) &&
+                Objects.equals(studyIdentifier, other.studyIdentifier) &&
+                Objects.equals(consentCreatedOn, other.consentCreatedOn) &&
+                Objects.equals(name, other.name) &&
+                Objects.equals(birthdate, other.birthdate) &&
+                Objects.equals(imageData, other.imageData) &&
+                Objects.equals(imageMimeType, other.imageMimeType) &&
+                Objects.equals(signedOn, other.signedOn) &&
+                Objects.equals(withdrewOn, other.withdrewOn) &&
+                Objects.equals(hasSignedActiveConsent, other.hasSignedActiveConsent));
+    }
+    
+    @Override
+    public String toString() {
+        return String.format("UserConsentHistory [healthCode=[REDACTED], studyIdentifier=%s, consentCreatedOn=%s, name=%s, birthdate=%s, imageData=%s, imageMimeType=%s, signedOn=%s, withdrewOn=%s, hasSignedActiveConsent=%s]",
+            studyIdentifier, consentCreatedOn, name, birthdate, imageData, imageMimeType, signedOn, withdrewOn, hasSignedActiveConsent);
```

Comment:
We also don't want to print the user's name or birthdate or their signature image data either.